src/Symfony/Component/EventDispatcher/Tests/ImmutableEventDispatcherTest.php

```diff
@@ -43,7 +43,7 @@ public function testDispatchDelegates()
         $this->innerDispatcher->expects($this->once())
             ->method('dispatch')
             ->with($event, 'event')
-            ->will($this->returnValue('result'));
+            ->willReturn('result');
```

Comment:
unrelated, let's save us some merge conflicts by reverting :)

Reply:
The fabbot complained. 🤷‍♂ 

But I could backport the change to 4.2 in order to keep merges easy. Or we ignore fabbot on this one. ;-)

Reply:
On lower branches then yes. And for the whole code base. Possible on 3.4 too?

Reply:
See #31738.

Reply:
Resolved after rebase.